.env.beta

```diff
@@ -1,4 +1,5 @@
 OPTIC_ASSEMBLED_SPEC_EVENTS=false
+OPTIC_SPLIT_SPEC_EVENTS=false
```

Comment:
New flag, as we still want the be able to turn the assembly (reading) and splitting (writing) on and off individually. In fact, `OPTIC_ASSEMBLED_SPEC_EVENTS` is actually only a flag for using the Rust implementation to read specs and handle commands, so perhaps a rename is in order.